### PR TITLE
Monitor utils hotfix

### PR DESCRIPTION
**Pull request by vkotronis**
### Description of PR

Hot-fix for ARTEMIS monitor utils.

What component(s) does this PR affect?

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [X] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs (incl. wiki)
- [X] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs (incl. wiki)
- [ ] Build System

### Related Issue

Resolves #

### Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [X] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vkotronis** [comment]
@slowr the PR breaks because the artemis-utils needs to be upgraded according to the changes of this branch (`__init__.py`, `LICENSE.txt`). Could you please upgrade accordingly to version 1.0.2 (and I will then re-run the PR).

File: utils/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (vkotronis) [inline utils/LICENSE.txt:1]: added the license as in the rest of the ARTEMIS code.

File: backend/core/detection.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 import pytricia
 import redis
 import ujson as json
+from artemis_utils import clean_as_path
```
Comment (vkotronis) [inline backend/core/detection.py:17]: moved this to artemis_utils to have the same code at one place.

File: backend/core/configuration.py [hunk 3]
```diff
@@ -10,13 +10,13 @@
...
 from typing import Text
 from typing import TextIO
-from typing import Tuple
 from typing import Union
 
 import redis
 import ruamel.yaml
 import ujson as json
 from artemis_utils import ArtemisError
+from artemis_utils import clean_as_path
```
Comment (vkotronis) [inline backend/core/configuration.py:19]: moved this to artemis_utils to have the same code at one place.

File: utils/artemis_utils/__init__.py [hunk 4]
```diff
@@ -278,6 +283,16 @@ def flatten(items, seqtypes=(list, tuple)):
     return res
 
 
+def load_json(filename):
```
Comment (vkotronis) [inline utils/artemis_utils/__init__.py:286]: added missing functions from the original monitor code utils (that file is deleted in the current PR).

File: utils/setup.py [hunk 5]
```diff
@@ -8,7 +8,7 @@
 
 setuptools.setup(
     name="artemis_utils",
-    version="1.0.1",
+    version="1.0.2",
```
Comment (vkotronis) [inline utils/setup.py:11]: this needs to be released as a new version with the changes (init code augmentation plus ARTEMIS license)

Comment (vkotronis) [reply utils/setup.py:11]: see https://pypi.org/project/artemis-utils/#history (should be updated)

**Review by slowr — APPROVED** [review]

**Pull request merged**